Makefile

```diff
@@ -86,7 +86,10 @@ mod-check: check-env
 	@echo 'running: go mod verify'
 	@go mod verify && [ "$(shell sha512sum go.mod)" = "`sha512sum go.mod`" ] || ( echo "ERROR: go.mod was modified by 'go mod verify'" && false )
 
-go-lint:
+scripts/golangci.yml: scripts/golangci.yml.in
+	sed "s/@@CEPH_VERSION@@/$(CEPH_VERSION)/g" < scripts/golangci.yml.in > scripts/golangci.yml
```

Comment:
can we remove this generated `scripts/golangci.yml` once the testing is done?

Reply:
We could, but there is no need to do so. If there are changes, to the template, it should get generated again.

Reply:
Well, maybe it should get removed with `make clean` just like the container images. I'll add that.